src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionMatchXpathTest.java

```diff
@@ -87,4 +88,154 @@ public void testTwo() throws Exception {
         runVerifications(moduleConfig, fileToProcess, expectedViolation,
                 expectedXpathQueries);
     }
+
+    @Test
+    public void testEncodedOne() throws Exception {
```

Comment:
Please also add test cases to SuppressionXpathFilterTest with escape sequences in file

Reply:
On it.

Reply:
Please see my comment. I am having some issues related to this teset.

Reply:
Done